website/docs/components/form/input-group/partials/specifications/anatomy.md

```diff
@@ -0,0 +1,26 @@
+## Anatomy
+
+### Base component
+
+![Input Group Base Anatomy](/assets/components/form/input-group/input-group-base-anatomy.png =800x*)
+
+| Element | Usage |
+|---------|-------|
+| Input | Required; can be one of many types extended via the [Text Input](/components/form/text-input) component. |
```

Comment:
There was an example on the "Guidelines" page that didn't include an Input; it only included Buttons/Toggles, so you may want to reconsider how this is documented.

Reply:
Within the base component this would be required, but using the primitives could create any type of custom combination. I'll look into adding a bit more clarity around the assembly of subcomponents and how the base component is related/builds upon.